lib/krane/cli/deploy_command.rb

```diff
@@ -54,6 +56,11 @@ def self.from_options(namespace, context, options)
           protected_namespaces = []
         end
 
+        options[:filenames] << "-" if options[:stdin]
```

Comment:
As discussed irl, we __must__ treat the `options` hash as `READ_ONLY`. Thus, maintain a local array for filenames instead of writing directly into `options[:filenames]`